server/controllers/imageController.js

```diff
@@ -78,9 +78,11 @@ const delete_image = async (req, res, next) => {
   const image_id = req.params.id;
   const user_id = req.user.user_id;
   const role = req.user.role;
-  console.log('Image id', image_id, user_id);
-  console.log('Id ', role);
   const deleted = await deleteImage(image_id, user_id, role, next);
+  if(!deleted){
+    res.json({ message: `Image deleted ${deleted}. Cannot delete others' image.` });
```

Comment:
image deleted and can not delete others image? lol

Reply:
image delete false ma ma